doc/dev/getting-started.rst

```diff
@@ -203,42 +202,32 @@ Claiming an issue and starting to develop
    changed. A file in the 'untracked' section are files that haven't existed
    previously in the repository but git has noticed.
 
-   To commit changes you have to 'stage' them--this is done by issuing the
+   To commit changes you have to 'stage' them—this is done by issuing the
    following command::
 
       git add path/to/file
 
-   If you have a large quantity of changes and you don't want to add each file
-   manually you can do ``git add --patch`` which will display each set of
-   changes to you before staging them for commit.
-
    Once you have staged your changes, it's time to make a commit::
 
-      git commit
+      git commit -m 'Here you provide a brief description of your changes'
```

Comment:
As a counterargument, consider the rotation students that, even after gentle consistent encouragement, are still hesitant and uncomfortable using a command-line text editor. This saves them the trauma of remembering `M^X` (nano) or `:wq` (vim) or `ctl-alt-shift-d-esc-tab-X` (or whatever crazy key combo emacs users use to quite).

This is also in line with the spirit of the rest of this PR: make the first contribution as painless as possible, but then gradually ask more of them as they make continued contributions.